tests/mock_tests/portsorch_ut.cpp

```diff
@@ -20,6 +20,32 @@ namespace portsorch_test
 
     using namespace std;
 
+    sai_queue_api_t ut_sai_queue_api;
+    sai_queue_api_t *pold_sai_queue_api;
+    int _sai_set_queue_attr_count = 0;
+
+    sai_status_t _ut_stub_sai_set_queue_attribute(
+        _In_ sai_object_id_t queue_id,
+        _In_ uint32_t attr_count,
+        _Inout_ sai_attribute_t *attr_list)
+    {
+        _sai_set_queue_attr_count++;
```

Comment:
please check for the specific sai attribute and bool value and then increment count.

Reply:
Done